DoSomething/quasar
# Adding Northstar source and updating docs and ref

#### What's this PR do?
This PR adds `norhtstar.users` as a source to DBT which allows it to be referenced in the DAG diagram. 

#### Where should the reviewer start?
#### How should this be manually tested?
The docs have been generated, but you should see this source pop up in the diagram once the GitHub pages have been built. The source itself can be tested while we build the `northstar_users_raw` table.

#### Any background context you want to provide?
#### What are the relevant tickets?
https://www.pivotaltracker.com/n/projects/2076969/stories/170879241
#### Screenshots (if appropriate)
#### Questions:
